src/main/java/cms/gov/madie/measure/services/TestCaseService.java

```diff
@@ -748,6 +741,25 @@ private TestCaseImportOutcome updateTestCaseJsonAndSaveTestCase(
             .patientId(testCaseImportRequest.getPatientId())
             .successful(false)
             .build();
+    if (appConfigService.isFlagEnabled(MadieFeatureFlag.LOCKING)) {
+      LockInfo lock =
+          testCaseLockService.lockTestCase(measureId, existingTestCase.getId(), userName);
+      log.info(
+          "User [{}] is trying to lock test case id: [{}]", userName, existingTestCase.getId());
```

Comment:
Since this is an import Test Cases event, in most cases users will be importing 10s of test cases at once, and if something fails, We need to look through logs based on measureId as we might not know which test case caused an issue. So I think it is beneficial to include measureId as well in the logs.

```suggestion
          "User [{}] is trying to lock test case id: [{}] for measureId: [{}]", userName, existingTestCase.getId(), measureId);
```

The same for other info logs as well.

Reply:
@RohitKandimalla thanks! that's a good point, I added measureId for those logging statements.